### ISSUE TITLE
Text labels in layout view

### ISSUE THREAD

**Issue by tbtgspence** [smeighan/xLights#2925]
It would be nice to be able to add text labels to the layout view that are merely there for organization and labeling of models.  

A use case is doing a large number of trees where we don't typically worry about arranging them visually like you would with a house.  We just line up all the trees in an area, but it would be nice to label them or label groups of them with text.

**Comment by bcarriveau** [comment]
![2021-11-10 18_59_05-xLights (Ver 2021 36 64bit) November 9, 2021](https://user-images.githubusercontent.com/60801063/141218015-30c244d8-036e-4fc8-bc6a-cc8f6fb56cb7.jpg)

**Comment by tbtgspence** [comment]
Looking for something more like this.  Would like to see the labels on the right and be able to position them so they can label multiple models.  Can't see how adding a description on the left would help in this situation.  In very large displays, the idea is to be able to easily recognize parts of the display by looking at the visual, not clicking on all the models individually.  I could see if creating a large drive through display with lots of elements, it could be nice to add labels to sections of the display for organization.

![Screenshot 2021-11-11 001912](https://user-images.githubusercontent.com/34466356/141263198-69183552-10ee-451d-9005-eb07273b1df2.png)

**Comment by keithsw1111** [comment]
I actually looked at this a long while back ... while I kind of got it working in 2D ... 3D was beyond me at the time.

**Comment by tbtgspence** [comment]
Yeah, I can certainly see the trouble with some of these features porting
across both types of views.

**Comment by cybercop23** [comment]
Any next steps here?

**Comment by AzGilrock** [comment]
I've thought about having a layer that you could draw on to represent things like controllers and wiring.  This text would be another candidate.  Should be designed so there's an easy way to toggle the layer on and off.

**Comment by cybercop23** [comment]
Several of us now use Images for Ctrl, Remotes, PI. Just place them where you need. I guess you can do what with text too, but you have to create an image.
